NEWS.md

```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
   intelmq.bots.experts.tor_nodes.expert --update-database
   ```
   The provided shell scripts use these new commands, however they are now deprecated and will be removed in version 3.0.
+- `intelmqctl list queues` is now able to sum events over all queues with the `--sum`, `--count` or simple `-s` flag.
 
 #### XMPP Bots
 Both the XMPP output bot and the XMPP collector bot are deprecated.
 The bots need to be migrate to another XMPP library (see [#1614](https://github.com/certtools/intelmq/issues/1614) for details) and a survey on the mailing listed revealed no users.
 If you depend on this bot, please reach out to us via the mailing list or GitHub.
 The bots are logging a deprecation warning now and the current plan is to remove them in IntelMQ version 3.0.
 
+#### Shadowserver Bots
+Shadowserver Collector is now able to collect data from [Shadowserver Reports API](https://intelmq.readthedocs.io/en/latest/user/bots.html#shadowserver-reports-api). Moreover Shadowserver Parser now supports JSON.
```

Comment:
**The** Shadowserver collector ... **the** Shadowserver Reports API... **the** Shadowserver parser now supports **the JSON format used by the Reports API**